[14.0][ADD] mrp_subcontracting_bom_dual_use: New addon. (#834)
OCA/manufacture: mrp_subcontracting_bom_dual_use/readme/DESCRIPTION.rst

> This module allows BOMs to be used for subcontracting and for normal production.

Reviewer: ```suggestion
This module allows subcontracting BOMs to be used for regular production orders as well.
```
